README.md

```diff
@@ -73,12 +73,12 @@ that content, run `grunt update`.
 #### Running the site for development and debugging
 
 1. `node scripts/update-server.js --console` will start the update server and
-log its output to the terminal window. This will *not* update wordpress, but
+log its output to the terminal window. This will *not* update Wordpress, but
 will let you see the result of adding a plugin locally.
 
 2. `node scripts/wordpress-update.js --console` will process the changes in
-sqlite into entries in wordpress. Note, if you're re-adding plugins that have
-already been added, you will need to remove those entries from wordpress.
+sqlite into entries in Wordpress. Note, if you're re-adding plugins that have
+already been added, you will need to remove those entries from Wordpress.
```

Comment:
It's WordPress, not Wordpress so both the old version and the new one is incorrect. Could you fix that?